Cassowary/ClLinearExpression.cs

```diff
@@ -394,8 +394,9 @@ public double NewSubject(ClAbstractVariable subject)
         ///      v1*c1 + v2*c2 + .. + vn*cn + c
         /// </summary>
         public double CoefficientFor(ClAbstractVariable var)
-        {
-            ClDouble coeff = _terms[var];
+        {
+            
+            ClDouble coeff = _terms.ContainsKey(var) ? _terms[var] : null;
```

Comment:
Rather than ContainsKey followed by getting the value, use [`TryGetValue`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=netcore-3.1) to do both operations at once.

Reply:
Good point. Fixed